virttest/libvirt_xml/vm_xml.py

```diff
@@ -2611,3 +2613,35 @@ def marshal_to_iothreads(tag, attr_dict, index, libvirtxml):
         if tag != 'iothread':
             return None
         return attr_dict['id']
+
+
+class VMFEATURESHPTXML(base.LibvirtXMLBase):
```

Comment:
class is all capital letter, I don't think it comply naming convention

Reply:
Updated